### DISCUSSION TITLE
Delete data from destination if it has been deleted from source

### DISCUSSION THREAD

**Discussion by Zakiirim** [estuary/flow#1265] [Ideas]
Hello, 

I'm right now testing a connector between Postgres (source) and Elasticsearch (destination). 

I noticed that if the data is deleted from the source, than the destination still retains the data but it's marked with `"_meta/op": "d"`.

Is there an option to just remove the associated deleted resource from the destination?

**Comment by dyaffe** [comment]
Hi @Zakiirim,

We are currently working on that functionality but it won't be ready for a little while.  If you'd like, I can follow up when it's available?  Our current estimate is 1-2 months.

Thanks,
Dave

**Reply by Zakiirim** [reply]
Hello @dyaffe, 

Thank you. Please keep me informed about that.

**Reply by alokranjanmagnifi** [reply]
any update here ? @dyaffe

**Comment by dyaffe** [comment]
@alokranjanmagnifi and @Zakiirim, the Elastic connector now supports a hard delete option.  Apologies that I forgot to follow up.
